onnx/onnx.proto

```diff
@@ -53,27 +53,49 @@ enum Version {
   // control. We should use version as
   //     xx(major) - xx(minor) - xxxx(bugfix)
   // and we are starting with 00000001.
-  IR_VERSION = 00000001;
+  IR_VERSION = 00000101;
```

Comment:
doesn't really match the comment?

Reply:
Fixed.  Also, I switched to hex constants, as using octal is pretty esoteric.